presto-bytecode/src/main/java/com/facebook/presto/bytecode/MethodDefinition.java

```diff
@@ -247,6 +271,12 @@ public void visit(ClassVisitor visitor, boolean addReturn)
             // visit instructions
             MethodGenerationContext generationContext = new MethodGenerationContext(methodVisitor);
             generationContext.enterScope(scope);
+
+            // To be safe, we estimate the code size here.
+            if (this.isFailOnLargeEstimatedSize() && estimateCodeSize(body) > MAX_ALLOWED_ESTIMATED_SIZE) {
+                throw new PrestoException(GENERATED_BYTECODE_TOO_LARGE, "Query results in large bytecode exceeding the limits imposed by JVM");
```

Comment:
Error string is used in two different places, so candidate for a constant?

Reply:
done

Reply:
This doesn't seem to have changed?